examples/dgmulti_2d/elixir_euler_triangulate_pkg_mesh.jl

```diff
@@ -13,11 +13,11 @@ meshIO = StartUpDG.triangulate_domain(StartUpDG.RectangularDomainWithHole())
 
 # the pre-defined Triangulate geometry in StartUpDG has integer boundary tags. this routine
 # assigns boundary faces based on these integer boundary tags.
-mesh = DGMultiMesh(dg, meshIO, Dict(:outer_boundary=>1, :inner_boundary=>2))
+mesh = DGMultiMesh(dg, meshIO, Dict(:outer_boundary => 1, :inner_boundary => 2))
 
 boundary_condition_convergence_test = BoundaryConditionDirichlet(initial_condition)
 boundary_conditions = (; :outer_boundary => boundary_condition_convergence_test,
-                         :inner_boundary => boundary_condition_convergence_test)
+                       :inner_boundary => boundary_condition_convergence_test)
```

Comment:
Would be nice to align the symbols if possible?

Reply:
@ranocha complained about this before; sadly, it does not seem to be possible 😢